src/blocks/author-list/components/inspector/AdvancedPanel.js

```
@@ -0,0 +1,99 @@
+						'--author-list-custom-var-2',
+						'author-profile-blocks',
+					) }
+					value={ customVar2 || '' }
+					onChange={ ( value ) => setAttributes( { customVar2: value } ) }
+				/>
+			</PanelBody>
+		</>
+	);
+};
+
+export default AdvancedPanel;
```

ESLint

[error] Newline required at end of file but not found.

(eol-last)